### DISCUSSION TITLE
沒達成計畫的煩惱

### DISCUSSION THREAD

**Discussion by aa103967622** [1101-GS4719-C/Python-Community-GS4719-C#1451] [General]
我總是有個美好的計畫，預計做這個預計做那個，把每天的行程排得相當充實，但有些事項總是不如我的預估，比如：安排某天下午空堂讀某科的兩個章節，但最後我可能只讀得完一個章節，然後就得去趕著上課了...像這樣安排的事情無法達到想像中的處理量，反而會讓人煩躁，更嚴重的，可能會犧牲睡眠時間去完成計畫，長久下來也積累不少睡眠債，讓健康每況愈下，不知道大家是否有相同的煩惱？是否有什麼方法可以解決呢？

**Comment by Alexgns** [comment]
如果每次皆無法完成是否考慮稍微將安排的工作份量降低，或找找看是否因其他原因分心導致無法完成，建議不要用睡眠時間來換，可能導致辦事效率越來越差

**Comment by alicia0725** [comment]
推薦你可以試試看將計畫分的更係然後可以計時看看比如說讀完一個章節大概要多久，在將一天中的時間排列出來看如何把事情安排進去，我自己還會多預留一個多餘的時間來補足做不完的事！

**Comment by zephyrxin01** [comment]
我覺得時間安排太緊湊可能會是可以調整的部分，你可以試試看在時間規劃時，預留一些時間是彈性時間，如果真的提早完成，可以當作是自己效率很高的休息小確幸。

**Comment by 1104030360** [comment]
我認為把計畫先從少到多開始排，不會排太細大概就好，然後把自己丟到一個能專注的環境確保完成計畫！

**Comment by sjh501** [comment]
我的習慣是沒有很詳細的進度，最多就只有當天的範圍，但大多數時間只有看有多少東西，不想碰的逼自己有一個進度而已。時間安排上，我習慣在空堂處理記憶性的、短時間的、或者沒有關連性的內容；長時間連續時間的則是讀有脈絡和需要投入大量時間的科目。如果當天完成不了會給自己一個最後休息時間，只要超過無論如何起床一早再處理。雖然很吃自我管理的能力就是了。

**Comment by pigeon9975** [comment]
真的有同感，每次排好行程後如果被莫名奇妙的事情打亂就會覺得很煩、很躁，然後就變成想要趕在短時間內做完好多好多事，回過頭來卻甚麼都讀不進去，甚麼都沒有完成，變得更煩、更躁，陷入惡性循環。所以我現在當自己意識到自己掉入這樣的迴圈時，就會先洗澡、吃飯，讓自己的腦袋冷靜下來，冷靜後再重新安排計畫，可能隔天早點起來，或是把比較不緊急的事往後移，讓所剩不多的時間能達到最好的應用。

**Comment by alstinlin** [comment]
我也常常會有這種感覺，比如說想在前一天晚上複習一下隔天上課進度，但突然有事導致時間被壓縮，但通常我發現是因為自己把時間排得太緊湊了，所以當我嘗試把時間安排的寬鬆一點就比較不會那麼有壓力，不過這樣做可能要犧牲一點娛樂的時間就是了

**Comment by ALAN30359704** [comment]
我也跟你一樣有相同的煩惱，但我之後有看過一個影片，他是在講說現在人對手機的依賴性非常的高，所以建議讀書的時候可以把手機放在不同的地方避免自己去滑手機，我認為這個是個有效可以達成計劃的方法。

**Comment by Elaine2899** [comment]
我也經常有這樣的煩惱，某天，我開始檢討自己會失敗的原因。我發現，一部份的原因是計畫不適合自己，我們容易將自己想的太完美。在計畫中，我建議加入適當的休息時間。另一部分則是自己會怠惰，時間到了卻不想跟著計劃走，這方面我建議將計畫安排的彈性一些，並且在一個項目之後安排獎勵自己的甜頭(但我自己最近是愛上解決代辦事項的成就感)。最後就是在每次失敗檢討一下這次為什麼失敗，並且記下來，或許是寫在書桌前、手機上等等，讓自己下次會盡量避開這些失敗的原因！

**Comment by Tzuhsuannnnn** [comment]
我很討厭在自己打算做甚麼事情時被突然打斷，且我認為人們很容易高估自己的效率，導致常常做不完預定要做的事，我之前都會每個小時每個小時安排要幹嘛，總覺得仔細安排比較能督促自己做事，但後來發現自己永遠都達不到預計要做完的事，因此我變成規劃好一天內多少時間應該要讀書或認真做甚麼事，而非規定甚麼時候要做甚麼，在規定的時間內讓自己專心做事，通常都比較有效率。

**Comment by Aaron0810** [comment]
這是真的，但我覺得身而為人可能這種事情沒辦法，除非你的毅力特別強大。而為了避免這種事情發生，我自己的話通常會把事情排的特別提早，假設我很清楚下禮拜要交甚麼作業，我會提早開始寫，但一天會排的少少的，因為我知道我會偷懶所以不敢排太多，會這樣說排少少的是因為我發現同學裡面真的有很多人喜歡把事情拖到最後一天，所以我覺得提早這件是很重要。

**Comment by xiang305** [comment]
我認為不用每天都給自己排一個很明確的進度，例如今天要讀完第幾章或第幾頁，而是改成今天預定要付出多少時間在那件事上，像是預計花一個小時讀微積分。這樣的話就不會讓自己那麼有壓力，甚至為了趕進度而犧牲了學習成效。之後再根據自己的進度調整每件事上要付出的時間，這樣的處理方式至少能讓我在deadline前完成該做的事。

**Comment by Evening218** [comment]
我贊同樓上大部分留言的想法,排計畫最重要的就是要預留彈性時間,避免一件事拖到進度就會影響後續所有的安排,甚至壓縮必要的休息時間,而且改成預定花多少時間在某件事上也是個很好的想法,我自己就是每天告訴自己要把固定的一段時間專心完成進度,只要時間到,就讓自己休息,這樣反而不會負擔太大,而且可以確實感受到自己有花時間做好該做的事

**Comment by poweichen00** [comment]
我也覺得不用特別給自己目標一定要達成多少進度，讓自己在時間內不會那麼有壓力，也能提高效率，這樣不會做起事來那麼緊湊。

**Comment by PaulSu0905** [comment]
我的方法是安排任務的優先次序，並留出緩衝時間。將大任務分解成小任務。練習有效的時間管理技巧，如波莫多羅技巧。記住要休息，優先考慮自我健康，並在需要時尋求幫助。

**Comment by larry40305** [comment]
每個人的情況不同，因此最重要的是找到適合自己的方法來解決這個問題。逐步調整計劃和心態，學會靈活應對，平衡工作和休息，將有助於減輕煩躁感並提高生活品質。
